JS.md

```diff
@@ -0,0 +1,237 @@
+# JavaScript API
+
+In the [MVP](MVP.md), the only way to access WebAssembly on the Web is through
+an explicit JS API which is defined below. (In the future, WebAssembly may also
+be loaded and run directly from an HTML `<script type='module'>` tag—and
+any other Web API that loads ES6 modules via URL—as part of 
+[ES6 Module integration](Modules.md#integration-with-es6-modules).)
+
+## The `Wasm` object
```

Comment:
(comment race) hah, right, so it's half initialism, half abbreviation, defying all classification.